main.go

```diff
@@ -44,7 +44,7 @@ var (
 	scheme               = runtime.NewScheme()
 	setupLog             = ctrl.Log.WithName("setup")
 	watchNamespaceEnvVar = "WATCH_NAMESPACE"
-	topologyPrefixes     = [...]string{"topology.kubernetes.io", "topology.block.csi.ibm.com"}
+	topologyPrefix       = "topology.block.csi.ibm.com"
```

Comment:
it can still be a list, with one item

Reply:
for what purpose?

Reply:
so it would be easy to add/remove prefixes

Reply:
do we have any intention of doing it?

Reply:
theoretically in the future (and definitely in this PR), sure

Reply:
ok, done